ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-base.yml

```
@@ -208,6 +216,14 @@ router_bgp:
         all: true
       # Testing bfd false OUTSIDE a peer group that should NOT render
       bfd: false
+      missing_policy:
+        direction_in:
+          action: deny
+          include_prefix_list: true
+          include_community_list: true
```

```suggestion
          include_community_list: true
          include_sub_route_map: true
```
let us add a test with all three include decisions as `true`